pkg/runtime/wfengine/component.go

```diff
@@ -122,6 +125,10 @@ func (c *workflowEngineComponent) Terminate(ctx context.Context, req *workflows.
 	}
 
 	if err := c.client.TerminateOrchestration(ctx, api.InstanceID(req.InstanceID), ""); err != nil {
+		if err == ErrInstanceNotFound {
```

Comment:
I think the correct way to compare errors in Go is to use `errors.Is(err, TheErrorImComparingTo)`